packages/editor/src/components/rich-text/index.js

```diff
@@ -413,10 +416,7 @@ export class RichText extends Component {
 		const record = this.createRecord();
 		const transformed = this.patterns.reduce( ( accumlator, transform ) => transform( accumlator ), record );
 
-		// Don't apply changes if there's no transform. Content will be up to
-		// date. In the future we could always let it flow back in the live DOM
-		// if there are no performance issues.
-		this.onChange( transformed, record === transformed );
+		this.onChange( transformed );
```

Comment:
This worries me that our standard set of operations for a single input have suddenly become more non-performant. Am I wrong to be worried?